pkg/cmd/release/upload/upload.go

```diff
@@ -122,3 +125,28 @@ func uploadRun(opts *UploadOptions) error {
 
 	return nil
 }
+
+func sanitizeFileName(name string) string {
+
+	value := regexp.MustCompile("[[:^ascii:]]").ReplaceAllLiteralString(name, "")
```

Comment:
This is not exactly what the platform does, since the platform transliterates non-ASCII Unicode characters to their Latin equivalents (e.g. converts `ć` in my name to `c`). We could pull in a transliteration library according to https://github.com/cli/cli/issues/7142#issuecomment-1465907659, or we could opt to do nothing in this case, since I'm guessing it's a rare circumstance that someone will use `--clobber` with file names that contains non-ASCII characters, but I don't think that replacing non-ASCII characters with nothing is useful here.

What do you think would be a good approach?

Reply:
BTW I just found transliteration rules that Rails uses; they look simple enough: https://github.com/ruby-i18n/i18n/blob/b4cd00d73234e369d713eb2607c729765bb3ec06/lib/i18n/backend/transliterator.rb#L44-L71

Reply:
Update: a transliteration effort in progress in another PR by a contributor: https://github.com/cli/cli/pull/7205#discussion_r1145429247

Reply:
@jtpetty Update: the function to remove diacritics from a string is now in the trunk branch: `text.RemoveDiacritics()`